consensus.go

```diff
@@ -221,11 +244,7 @@ func (p *Pbft) SetBackend(backend Backend) error {
 
 // start starts the PBFT consensus state machine
 func (p *Pbft) Run(ctx context.Context) {
-	p.ctx = ctx
-
-	// the iteration always starts with the AcceptState.
-	// AcceptState stages will reset the rest of the message queues.
-	p.setState(AcceptState)
+	p.RunPrepare(ctx)
```

Comment:
Since this method is making initial state of the state machine, maybe we can rename it?